Recrutify/Recrutify.DataAccess/Configuration/MailSettings.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Recrutify.DataAccess.Configuration
```

Comment:
email models are not related to DataAccess layer, so replace them all to Host project, and remove Configuration folder here